src/locales/en-US/commands/choice.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "choice",
+	"description": "Get a random value from a set of choices",
+	"optionsValuesName": "values",
+	"optionsValuesDescription": "The choices to pick a random value from, separated by spaces",
+	"tooFewOptions": "There needs to be at least two values.",
+	"duplicatedOptions": "There is at least one duplicate value in your set, please remove them.",
+	"result": "👯 *Eeny, meeny, miny, moe, catch a tiger by the toe*... I choose: ``{{value}}``"
```

Comment:
Don't ask me what else but we may need to change this because of its racist history: https://aninjusticemag.com/the-racist-origin-of-eeny-meeny-miny-moe-411e2d4815ac

Reply:
Give me an alternative, then.